pkg/process/exec_conf.go

```diff
@@ -128,10 +135,6 @@ func cleanup(cmd *cobra.Command) {
 		defer mon.TaskNamed("root")(&ctx)(&err)
 
 		vip := viper.New()
-		err = vip.BindPFlags(cmd.Flags())
```

Comment:
Why remove the binding of flags to viper?

Reply:
This call makes the `Changed` field of every flag to be `true`. Which is problematic for the SaveConfig.

Reply:
But, then, doesn't this break the viper functionality? Are the flags still set somehow to the values in the config file that viper reads?

Reply:
You got a point. We reverted this line and found another solution for restoring the `Changed` field.